appinventor/components-ios/src/File.swift

```diff
@@ -122,18 +122,144 @@ open class File: NonvisibleComponent {
         DispatchQueue.main.async {
           if append {
             self._form?.dispatchErrorOccurredEvent(self, "AppendTo",
-                ErrorMessage.ERROR_CANNOT_CREATE_FILE.code,
-                ErrorMessage.ERROR_CANNOT_CREATE_FILE.message, fileName)
+                                                   ErrorMessage.ERROR_CANNOT_CREATE_FILE.code,
+                                                   ErrorMessage.ERROR_CANNOT_CREATE_FILE.message, fileName)
           } else {
             self._form?.dispatchErrorOccurredEvent(self, "SaveFile",
-                ErrorMessage.ERROR_CANNOT_CREATE_FILE.code,
-                ErrorMessage.ERROR_CANNOT_CREATE_FILE.message, fileName)
+                                                   ErrorMessage.ERROR_CANNOT_CREATE_FILE.code,
+                                                   ErrorMessage.ERROR_CANNOT_CREATE_FILE.message, fileName)
           }
         }
       }
     }
   }
   
+  // Copy the contents from the first file to the second file.
+  @objc func CopyFile(_ fromPath: String, _ toPath: String) -> Bool {
```

Comment:
@dhruvshrivastava18 @barreeeiroo We need to think about what the right approach is here because the App Inventor signature for CopyFile is different from the Kodular one, and it's not clear exactly to me what the upgrade path should be.

Reply:
The signature between Kodular and AI2 is actually the same. The difference is the scope, that we are not really sure how we can handle and map it to iOS correctly.